app/js/components/mixins/ModalEffects.react.js

```diff
@@ -1,23 +1,40 @@
 import React from 'react'
 
+/*
+  Here's the ultimate modal creator :)
```

Comment:
👍 .. Just some few thoughts. 
This part might need to be refactored. As I've re-iterated earlier since we are only using this component for Tagmodal using this as a mixin might be a little bit too early.

MountAsModal method needs to be refactored.

We should avoid using timeout without having it set in a variable so we can destroy it to avoid memory leaks.

Apart from that this looks good.

PS if we are to take care of that setTimeout This component can turn to pure functional component Which is better